src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -66,6 +67,7 @@ case class BenchmarkResult(
     optimizationTime: Option[Double] = None,
     planningTime: Option[Double] = None,
     executionTime: Option[Double] = None,
+    hashSum: Option[Long] = None,
```

Comment:
maybe just make this `result`? I was thinking other execution modes might do things like return num rows. (though then maybe we should also include the execution mode in the result?)